backend/config/database.js

```diff
@@ -4,7 +4,7 @@ module.exports = ({ env }) => {
   if (env("NODE_ENV") === "production") {
     return {
       connection: {
-        client: "postgres",
+        client: "mysql",
```

Comment:
For now, lets just allow overriding Postgres from the ENV?
```suggestion
        client: env("DATABASE_ENGINE", "postgres"),
```